Block/Adminhtml/System/Config/Field/SelectYesNo.php

```diff
@@ -15,24 +15,29 @@
 
 class SelectYesNo extends Select
 {
-    /**
-     * Options
-     *
-     * @var array
-     */
-    protected $options = [
-        '0' => 'No',
-        '1' => 'Yes'
-    ];
+    private function getSourceOptions(): array
+    {
+        return [
+            ['label' => 'No', 'value' => '0'],
+            ['label' => 'Yes', 'value' => '1'],
+        ];
+    }
+
+    public function _toHtml(): string
+    {
+        if (!$this->getOptions()) {
+            $this->setOptions($this->getSourceOptions());
+        }
```

Comment:
same :)

Reply:
I would rather keep it here because that class is an extension of `Magento\Framework\View\Element\Html` and this method will be called specifically. What do you think?